src/lib.rs

```diff
@@ -316,13 +316,21 @@ impl Database {
         // Linux.
         #[cfg(all(unix, not(any(target_os = "macos", target_os = "android"))))]
         {
+            let home = std::env::var("HOME").unwrap_or("~".to_string());
```

Comment:
This should be `.unwrap_or_else(|| "~".to_string());`.

Reply:
Or actually, we should not do this either. If `HOME` is not set - bail immediately.